### PR TITLE
workflows: fixed accesses to authors api

### PR DESCRIPTION
**Pull request by DonHaul**
* ref: cern-sis/issues-inspire/issues/614

### CODE REVIEW

File: workflows/plugins/hooks/backoffice/workflow_management_hook.py [hunk 1]
```diff
@@ -16,8 +16,12 @@ class WorkflowManagementHook(BackofficeHook):
     :type http_conn_id: str
     """
 
+    def __init__(self,collection):
```
Comment (drjova) [inline workflows/plugins/hooks/backoffice/workflow_management_hook.py:19]:
```suggestion
    def __init__(self, collection):
```

File: workflows/plugins/hooks/backoffice/workflow_management_hook.py [hunk 2]
```diff
@@ -16,8 +16,12 @@ class WorkflowManagementHook(BackofficeHook):
...
+        super().__init__()
+        self.collection = collection
+
```
Comment (drjova) [inline workflows/plugins/hooks/backoffice/workflow_management_hook.py:22]: could you please also add tests for that?

File: workflows/dags/author/author_create/author_create_init.py [hunk 3]
```diff
@@ -86,9 +83,7 @@ def create_author_create_user_ticket(**context: dict) -> None:
     def set_author_create_workflow_status_to_approval(**context: dict) -> None:
         status_name = "approval"
         workflow_management_hook.set_workflow_status(
-            status_name=status_name,
-            workflow_id=context["params"]["workflow_id"],
-            collection=AUTHORS,
+            status_name=status_name, workflow_id=context["params"]["workflow_id"]
```
Comment (drjova) [inline workflows/dags/author/author_create/author_create_init.py:86]: why on the same line?

File: workflows/dags/author/author_create/author_create_rejected.py [hunk 4]
```diff
@@ -28,24 +28,20 @@ def author_create_rejected_dag() -> None:
...
+    workflow_management_hook = WorkflowManagementHook(AUTHORS)
 
     @task()
     def set_author_create_workflow_status_to_completed(**context: dict) -> None:
         status_name = "completed"
         workflow_management_hook.set_workflow_status(
-            status_name=status_name,
-            workflow_id=context["params"]["workflow_id"],
-            collection=AUTHORS,
+            status_name=status_name, workflow_id=context["params"]["workflow_id"]
```
Comment (drjova) [inline workflows/dags/author/author_create/author_create_rejected.py:37]: have you enabled pre-commit? 🤔

File: workflows/tests/cassettes/TestWorkflowManagementHook.test_get_workflow_url.yaml [hunk 5]
```diff
@@ -0,0 +1,442 @@
...
+      Accept-Encoding:
+      - gzip, deflate
+      Authorization:
+      - Token 2e04111a61e8f5ba6ecec52af21bbb9e81732085
+      Connection:
+      - keep-alive
+      User-Agent:
+      - python-requests/2.31.0
+    method: GET
+    uri: http://host.docker.internal:8001/api/workflows/authors/invalid_workflow_id/
```
Comment (drjova) [inline workflows/tests/cassettes/TestWorkflowManagementHook.test_get_workflow_url.yaml:60]: we have to ignore this host as well, for now the pyvcr only ignores localhost

**Review by drjova — CHANGES_REQUESTED** [review]
Thanks @DonHaul, few comments

File: workflows/dags/author/author_create/author_create_rejected.py (hunk 4 above)
Comment (DonHaul) [reply workflows/dags/author/author_create/author_create_rejected.py:37]: yes. It was not working I fixed it and reran it. I believe it was it that made this change

**Review by drjova — APPROVED** [review]

**Pull request merged**